### ISSUE TITLE
Still getting issue with rsync

### ISSUE THREAD

**Issue by DCossey** [DerrickWood/kraken2#797]
Hi everyone, 

I am still getting the issue below when trying to build the bacteria kraken database from NCBI.

rsync_from_ncbi.pl: unexpected FTP path (new server?) for https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/900/128/725/GCF_900128725.1_BCifornacula_v1.0

I saw many previous discussions about this issue and tried the suggestions to edit line 46 of rsync_from_ncbi.pl. But for me the line is already like this:  

if (! ($full_path =~ s#^(?:ftp|https)://${qm_server}${qm_server_path}/##)) {

Which makes me think that the code has already been updated to account for the https/ftp problems. However, it still does not work for me. I tried to change it to this suggestion (https://github.com/DerrickWood/kraken2/issues/518#issuecomment-1496518813) but it did not work. Any help would be much appreciated. 

Thanks

**Comment by Somebodyatthdoor** [comment]
We are having the same problem. Only seems to be for the bacteria database.

**Comment by tibitoy** [comment]
I find that by downloading the prebuilt version and uncompressing the library and taxonomy directories are not created and so using tools like Bracken is impossible. Have you found a way to circumvent that @rawosemo ?

**Comment by rawosemo** [comment]
@tibitoy, I have used Bracken recently. For me, I don't have the library and taxonomy directories created, but I was able to use Bracken.

Are you getting a specific error code?

**Comment by tibitoy** [comment]
Yep, I'm getting the following:  ERROR: Database taxonomy /home4/tibitoy/C-WAP/kraken_db/taxonomy/nodes.dmp does not exist

**Comment by rawosemo** [comment]
Oh! It can't find the “nodes.dmp” file within the directory you specified. 

There is no “taxonomy” directory if you used the pre-built version. You might want to remove it from your file path. 

For example, I just specify the path to the database whenever I want to run any analysis.

/mnt/data/kraken_db

I don't add taxonomy or library.  And I have recently used Bracken without facing such challenge.

**Comment by tibitoy** [comment]
Thanks! How do I go about removing it from my file path? I specific my path as well /home4/tibitoy/C-WAP/kraken_db and that contains the pre-build database but I still get the above error.
